src/js/view/controls/play-display-icon.js

```diff
@@ -15,34 +17,53 @@ export default class PlayDisplayIcon {
             this.trigger(evt.type);
         });
 
-        _model.on('change:state', (model, newstate) => {
-            let newstateLabel;
-            switch (newstate) {
+        _model.on('change:state', (model, newState, oldState) => {
+            let newStateLabel;
+            switch (newState) {
                 case 'buffering':
-                    newstateLabel = localization.buffer;
+                    newStateLabel = localization.buffer;
                     break;
                 case 'playing':
-                    newstateLabel = localization.pause;
+                    newStateLabel = localization.pause;
                     break;
                 case 'paused':
-                    newstateLabel = localization.playback;
+                    newStateLabel = localization.playback;
                     break;
                 case 'complete':
-                    newstateLabel = localization.replay;
+                    newStateLabel = localization.replay;
+                    break;
+                case 'idle':
+                    newStateLabel = localization.playback;
                     break;
                 default:
-                    newstateLabel = '';
+                    newStateLabel = '';
                     break;
             }
-            if (newstateLabel === '') {
+            if (newStateLabel === '') {
                 iconDisplay.removeAttribute('aria-label');
             } else {
-                iconDisplay.setAttribute('aria-label', newstateLabel);
+                iconDisplay.setAttribute('aria-label', newStateLabel);
```

Comment:
The else seems like the more common outcome here. This could be simplified:
```
iconDisplay.setAttribute('aria-label', newStateLabel);

if (newStateLabel === '') {
    iconDisplay.removeAttribute('aria-label');
}
```

Reply:
No reason to set and then remove. else if good, but maybe the order should be reversed?